### PR TITLE
added tina-init-tutorial.md

### PR DESCRIPTION
**Pull request by logan-anderson**
This is the getting started tutorial for Tina init.

This will not be linked on our site but will be used in an i frame in the users app. 

Link: https://tinacms-site-next-git-feat-tina-init-tutorial-tinacms.vercel.app/docs/tina-init-tutorial/?layout=false

This also adds the ability to remove the layout when in the docs with a query param. (tina.io/docs/some-doc-page/?layout=false). This will only render the doc page and not the nav, header, footer, etc.

### CODE REVIEW

File: content/docs/tina-init-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+To hook up this demo to Tina Cloud and save content to Github instead of the file system you can do the following.
+
+1.  Register at https://auth.tina.io
+2.  Update .env file to include:
+
+```
+NEXT_PUBLIC_ORGANIZATION_NAME= get this from the organization you create at auth.tina.io
+NEXT_PUBLIC_TINA_CLIENT_ID= get this from the app you create at auth.tina.io
+NEXT_PUBLIC_USE_LOCAL_CLIENT=0
+å```
```
Comment (perkinsjr) [inline content/docs/tina-init-tutorial.md:221]: remove the artifact that appeared.

File: content/docs/tina-init-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
+Hello! and thanks for bootstrapping a Tina App! Before you do anything click on "toggle edit state" button and a pencil icon in the bottom left hand corner will appear. You can now edit this content in real time! Click save and notice that you have update the Hello world blog post in the local file system.
+
+## Next steps
+
+### Wrap your App in "Edit State" (**Can probably skip this as CLI did this for you**)
```
Comment (perkinsjr) [inline content/docs/tina-init-tutorial.md:5]: Maybe we can remove the () part and place a quick paragraph that explains if you bootstrapped the application we have taken care of it for you.

File: content/docs/tina-init-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+To make a page editable we need to do three things
+
+1.  Update our schema.ts
+2.  Update our query
+3.  Update our code
+
+#### Update schema.ts
+
+lets update our schema.ts to include product listings. We will do so by added an "pages collection" with a product listing template. When added to our existing blog collection it looks like this. We also need a place to store the authors in the file system. So lets create a folder "content/pages".
```
Comment (perkinsjr) [inline content/docs/tina-init-tutorial.md:76]: Just some grammar (added) should be adding

File: content/docs/tina-init-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+```md
+---
+_template: product
+---
+```
+
+
+#### Make a new Next.js page and a graphql Query
+
+Start by making a new file called pages/product-listing.tsx that contains the following
```
Comment (perkinsjr) [inline content/docs/tina-init-tutorial.md:147]: should be a code span for pages/product-listing.tsx

File: content/docs/tina-init-tutorial.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+        variables: {},
+      }),
+      variables: {},
+      query,
+    },
+  };
+};
+```
+
+We are doing a couple of things here. A graphql query was written that looks for the content/pages/product-listing.md file
```
Comment (perkinsjr) [inline content/docs/tina-init-tutorial.md:202]: code span for content/pages/product-listing.md

File: content/docs/tina-init-tutorial.md [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+
+The query is being statically exported so it can be accessed by the TinaWrapper.
+
+The code was updated to display the list of products.
+
+Visit [http://localhost:3000/product-listing{" "}](http://localhost:3000/product-listing) {" "} to see what you just created. Click the "edit this site" button in the top to edit.
```
Comment (perkinsjr) [inline content/docs/tina-init-tutorial.md:208]: The editor has added {" "} we should remove this.

**Review by perkinsjr — CHANGES_REQUESTED** [review]
Overall this looks amazing, just a few nit style changes.

File: content/docs/tina-init-tutorial.md (hunk 6 above)
Comment (logan-anderson) [reply content/docs/tina-init-tutorial.md:208]: This is me converting JSX to markdown 😄

**Comment by perkinsjr** [comment]
Everything looks good, we need to update lets to Let's throughout but apart from that we should be good to go

File: content/docs/tina-init-tutorial.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+
+To make a page editable we need to do three things
+
+1.  Update our schema.ts
+2.  Update our query
+3.  Update our code
+
+#### Update schema.ts
+
+lets update our schema.ts to include product listings. We will do so by adding a "pages collection" with a product listing template. When adding to our existing blog collection it looks like this. We also need a place to store the authors in the file system. So lets create a folder `content/pages`.
```
Comment (perkinsjr) [inline content/docs/tina-init-tutorial.md:78]: Make this Let's

File: content/docs/tina-init-tutorial.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+          ],
+        },
+      ],
+    },
+  ],
+});
+```
+
+
+Lets also create a file to query. In the folder we just created (`content/pages`) add a file called `product-listing.md`
```
Comment (perkinsjr) [inline content/docs/tina-init-tutorial.md:138]: Let's

**Review by perkinsjr — COMMENTED** [review]
Just two spots and we should be good

**Review by perkinsjr — APPROVED** [review]

**Comment by perkinsjr** [comment]
LGTM Merge when ready

**Review by Enigmatical — APPROVED** [review]
👍

**Pull request merged**